service/scaladsl/api/src/main/scala/com/lightbend/lagom/scaladsl/api/LagomConfigComponent.scala

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.scaladsl.api
+
+import com.typesafe.config.Config
+import play.api.Configuration
+
+trait LagomConfigComponent {
+  def configuration: Configuration
+  def config: Config = configuration.underlying
+}
```

Comment:
When are you thinking we'd make that change? Before merging this or at some point in the future?